### PR TITLE
Check html semantics and stlying

### PR DESCRIPTION
**Pull request by inspecta**
This is a portfolio mobile first replica of a web template at Figma.
I have included linters, images, stylesheets, boxicons and a project license.
Checkout my HTML and CSS writing, use of HTML semantics and position of elements plus my stylings.

### CODE REVIEW

**Comment by BertrandConxy** [comment]
Hello @inspecta Please make sure that all linters are passing 😃 👍

**Review by BertrandConxy — COMMENTED** [review]
Hi @inspecta , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that has some linter errors


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          me.
+        </p>
+      </main>
+      <footer>
+        <h4 class="social-title">LET'S CONNECT</h4>
+        <div class="social-icons">
+          <a href="#"><i class="bx bxl-twitter"></i></a>
+          <a href="#"><i class="bx bxl-linkedin"></i></a>
+          <a href="#"><i class="bx bxl-github"></i></a>
+        </div>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly use an unordered list to list social icons. This is best practice for HTML semantics.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <link
+      href="https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="logo-placeholder">
+          <p class="logo">M.D</p>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly consider using an `<a>` to wrap your logo. This is best practice and it is [standard convention](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </div>
+        <nav class="nav-links">
+          <img class="menu-btn" src="/images/menu-btn.svg" alt="Menu Button" />
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly check the link to your hamburger icon as the icon doesn't display on the webpage.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      </header>
+      <main>
+        <h1 class="title">I'm Derrick. Nice to see you!</h1>
+        <p class="about-me">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+      </main>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly use a `<section>` element to wrap the headline section as this is convention for HTML5 semantics.

- [ ] Please add the background image to this section as it is in the template.

File: index.html (hunk 1 above)
Comment (Kamba56) [inline index.html]: - [ ] Please note that this is also part of the headline section. Kindly move it to the section.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @inspecta, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] linters passed all checks
- [x] Professional Readme
- [x] Descriptive PR description

## Required Changes ♻️
_Please check the comments under the review for suggested changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -15,29 +15,35 @@
...
-      </main>
-      <footer>
+      </section>
+      <section class="footer">
         <h4 class="social-title">LET'S CONNECT</h4>
-        <div class="social-icons">
-          <a href="#"><i class="bx bxl-twitter"></i></a>
-          <a href="#"><i class="bx bxl-linkedin"></i></a>
-          <a href="#"><i class="bx bxl-github"></i></a>
-        </div>
```
Comment (Kamba56) [inline index.html]: - [ ] As requested by the previous reviewer, kindly add a background image to this section to match the design template.
*Your project*ccc
*Design template*aaa

- [ ] Kindly change your alignments to match the design template. Please note that your project design should look exactly like the Figma template you chose.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @inspecta, 

Great work on making the changes requested by a previous reviewer 👏
 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Meanignful commit messages
- [x] All linters checks passed
- [x] Descriptive PR summary

 ## Required Changes ♻️
- Please in all future projects, kindly make sure that your branch name should have a name of a feature or action implemented in the branch and your PR title should be a short summary of the changes implemented in the PR.
 
_Kindly check the comments under the review for suggested changes._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @inspecta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**